graphics/rendering/blender/README.md

```diff
@@ -2,7 +2,7 @@
 
 Blender is a free and open source software used for editing and rendering.
 
-In this example, we will install Blender in a [Ubuntu container](https://cloud.sylabs.io/library/library/default/ubuntu).
+In this example, we will run Blender in a [Ubuntu container 16.04](https://cloud.sylabs.io/library/library/default/ubuntu).
```

Comment:
change this from run to "install and run" Blender

Reply:
:+1: